pdfminer/high_level.py

```diff
@@ -88,3 +90,46 @@ def extract_text_to_fp(inf, outfp,
         interpreter.process_page(page)    
 
     device.close()
+
+
+def pdf_to_text(pdf_file, password="", page_numbers=set(), maxpages=0,
+                caching=True, codec="utf-8"):
```

Comment:
I like that this function has only a minimal number of arguments. But since the `LAParams` is the thing you want to change most often, I think we should include that.

Reply:
I've personally never messed with LAParams to be quite honest, that's why I didn't include it. Should I add it as the last parameter, or somewhere before the other?

Reply:
I dont really have an opinion on where they should be added.

Reply:
Added it as the last parameter